File: Algoritma/Perulangan Bilangan Prima/Perulangan-Bilangan-Prima.py
batas = int (input ("Masukkan batas perulangan bilangan prima: "))
print (f"Bilangan prima dari 2 hingga {batas}:")

for i in range (2, batas + 1):
    prima = True

    for j in range (2, int (i ** 0.5) + 1):
        if i % j == 0:
            prima = False
            break
    if prima:
        print (i, end = " ")
print ()

File: Algoritma/Perulangan Bilangan Prima/PerulanganBilanganPrima.java
import java.util.Scanner;

public class PerulanganBilanganPrima
{
    public static void main (String [] args)
    {
        Scanner scan = new Scanner (System.in);
        System.out.print ("Masukkan batas perulangan bilangan prima: ");
        int batas = scan.nextInt ();

        System.out.printf ("Bilangan prima dari 2 hingga %d:%n", batas);

        for (int i = 2; i <= batas; i++)
        {
            boolean prima = true;

            for (int j = 2; j * j <= i; j++)
            {
                if (i % j == 0)
                {
                    prima = false;
                    break;
                }
            }
            if (prima)
                System.out.print (i + " ");
        }
        System.out.println ();
    }
}